app/views/participant_signup/_contact.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= f.inputs :name => "Contact Information" do -%>
```

Comment:
One possibility is to use the HelpText model I created. It is a fully dynamic way of storing help text and hints for any element on a per-tenant basis. This allows CAN to use a different help text for a field than the UWDP, for example. We could possibly use that, with some modifications, if the main difference is primarily help text. It could be modified to handle different "required" states too I suppose. 

https://github.com/uwdreamproject/DreamSIS/blob/master/app/models/help_text.rb

FYI there is currently no UI for this - but that could be created and allow staff to update the text in the mentor UI without requiring code changes...